src/util.cpp

```diff
@@ -590,6 +590,10 @@ fs::path GetWalletDir()
         }
     } else {
         path = GetDataDir();
+        // If a wallets directory exists, use that, otherwise default to GetDataDir
+        if (fs::is_directory(path / "wallets")) {
```

Comment:
In commit *Default wallet directory is wallets/ if it exists*, missing test?

Reply:
This is part of the WIP, I'll add a test for that when I make it actually initialise the directory :)